src/DB/PageIndex.php

```diff
@@ -169,41 +200,77 @@ private static function getStaticRows(): array {
+### Franky ![Franky](src/images/site/fun-robot.png)
+
+Franky was our first attempt at an autonomous digging robot. He got his name as more parts were grafted onto the body as the "features" evolved.
 EOT
          ],
          ['type' => self::DEFAULT_TYPE,
-          'theme' => self::PURPLE_THEME,
+          'theme' => self::GREEN_THEME,
           'pageid' => 4,
-          'page_title' => 'Test 4 - Website Demo',
-          'page_header' => 'Test Page 4',
-          'nav_text' => null,
+          'page_title' => 'Life - Website Demo',
+          'page_header' => 'Life',
+          'hide_main_nav' => true,
+          'main_article' => ''
+         ],
+      ];
+   }
+
+   private static function getLiveStaticRows(): array {
+      return [
+         ['type' => self::HOMEPAGE_TYPE,
+          'theme' => self::ORANGE_THEME,
+          'pageid' => 1,
+          'page_title' => 'About Me - Website Demo',
+          'page_header' => 'About Me',
+          'nav_text' => 'You might also enjoy...',
+          'main_article' => <<<EOT
+## This is the About Me Article!
+
+I write code and don't have _any_ coding examples. I hope this will serve both as my personal website and an example of how I write code!
+EOT
+         ],
+         ['type' => self::WORK_TYPE,
+          'theme' => self::GREY_THEME,
+          'pageid' => 2,
+          'page_title' => 'Work - Website Demo',
+          'page_header' => 'Work',
+         ],
+         ['type' => self::DEFAULT_TYPE,
+          'theme' => self::BLACK_THEME,
+          'pageid' => 3,
+          'page_title' => 'Robots - Website Demo',
+          'page_header' => 'Robots',
           'main_article' => <<<EOT
-## This is _Test Page 4_
+## Robots
 
-Vulputate dignissim suspendisse in est. Amet risus nullam eget felis eget nunc lobortis. Pellentesque diam volutpat commodo sed egestas. Id leo in vitae turpis massa sed elementum tempus egestas. Nam libero justo laoreet sit amet cursus sit. Consectetur purus ut faucibus pulvinar. Laoreet suspendisse interdum consectetur libero id faucibus nisl. Laoreet non curabitur gravida arcu ac tortor dignissim convallis aenean. Viverra mauris in aliquam sem fringilla. Nibh nisl condimentum id venenatis a condimentum vitae sapien pellentesque. Ut placerat orci nulla pellentesque. Bibendum at varius vel pharetra vel turpis nunc eget lorem. Euismod quis viverra nibh cras pulvinar mattis nunc sed.
+### Franky ![Franky](https://storage.googleapis.com/burnished-flare-348022.appspot.com/images/site/c2d2.png)
```

Comment:
Oops... Nbd b/c we'll change the text later but want to be aware of making sure to not use the site id.